### PR TITLE
New package: parity-1.8.6

### PR DESCRIPTION
**Pull request by wpbirney**

### CODE REVIEW

File: srcpkgs/parity/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+hostmakedepends="cargo pkg-config"
+makedepends="eudev-libudev-devel libressl-devel"
+short_desc="Fast, light, robust Ethereum implementation"
+maintainer="Wilson Birney <wpb@360scada.com>"
+license="GPL-3"
+homepage="https://github.com/paritytech/parity/"
+distfiles="https://github.com/paritytech/parity/archive/v${version}.tar.gz"
+checksum=0923b18ff2f52d3a6144d587cef0e489abb1308844c525b8fb86d4150134c320
+nocross=yes
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/parity/template]: no

File: srcpkgs/parity/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+system_accounts="parity"
```
Comment (Vaelatern) [inline srcpkgs/parity/template]: Start system name with an underscore.

File: srcpkgs/parity/INSTALL.msg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+After enabling the parity service
+
+visit web3.site in your browser to access parity
```
Comment (Vaelatern) [inline srcpkgs/parity/INSTALL.msg]: remove file

File: srcpkgs/parity/template (hunk 1 above)
Comment (wpbirney) [reply srcpkgs/parity/template]: @Vaelatern it fails to build on any other arch including musl

**Comment by wpbirney** [comment]
i previously had `only_for_archs="i686 x86_64"` because it fails to build on musl..

**Comment by newbluemoon** [comment]
> it fails to build on musl

If you can’t fix it you can use something like:

```
case "$XBPS_TARGET_MACHINE" in
        *-musl) broken="<url to build log>";;
esac

```

**Comment by wpbirney** [comment]
@newbluemoon thank you, i was not sure of the proper approach when a package fails to build on musl

**Pull request closed without merging**